rs/cross-chain/proposal-cli/src/canister/mod.rs

```diff
@@ -46,13 +46,13 @@ impl TargetCanister {
     pub fn candid_file(&self) -> PathBuf {
         match &self {
             TargetCanister::CkBtcArchive | TargetCanister::CkEthArchive => {
-                PathBuf::from("rs/rosetta-api/icrc1/archive/archive.did")
+                PathBuf::from("rs/ledger_suite/icrc1/archive/archive.did")
```

Comment:
@mbjorkqvist @NikolasHai : One drawback here is that if one were to try to make a proposal for a commit that pre-dates this moving, then this will fail because the tool will look for a file `rs/ledger_suite/icrc1/archive/archive.did` which is actually at `rs/rosetta-api/icrc1/archive/archive.did`. Solving this would require a bit more work (one would need to determine if the target commit is before or after this move) but I have the impression that it's not worth it ( we usually make upgrade proposals for new versions, not commit that are a lot in the past). In the worst case, one would need to use this tool at the same commit one wants to upgrade the canister to. WDYT?

Reply:
Good catch, thanks @gregorydemay! Another issue is with the git log command in the proposal cli that would for the next version(s) probably need to look in both places?

Reply:
> Another issue is with the git log command in the proposal cli that would for the next version(s) probably need to look in both places?

yes, that's also a good point. Also here I have the impression that this would be a one-off issue so doing the proper solution here might not be worth it.

Reply:
What are that odds that we try to upgrade to a commit that predates the commit after this merge?

Reply:
> What are the odds that we try to upgrade to a commit that predates the commit after this merge?

I think this is unlikely, but it will impact the next round of proposals for all canisters supported by this tool, which are just not XC canisters but also FI canisters, that's why I wanted to raise the issue. But I agree that this should be a one-off problem and so I don't think we need anything fancy.